python/paddle/fluid/evaluator.py

```diff
@@ -285,3 +287,120 @@ def eval(self, executor, eval_program=None):
             result = executor.run(
                 eval_program, fetch_list=[avg_distance, avg_instance_error])
         return np.array(result[0]), np.array(result[1])
+
+
+class DetectionMAP(Evaluator):
+    """
+    Calculate the detection mean average precision (mAP).
+
+    TODO (Dang Qingqing): update the following doc.
+    The general steps are as follows:
+    1. calculate the true positive and false positive according to the input
+        of detection and labels.
+    2. calculate mAP value, support two methods: '11 point' and 'integral'.
+
+    Please get more information from the following articles:
+      https://sanchom.wordpress.com/tag/average-precision/
+      https://arxiv.org/abs/1512.02325
+
+    Args:
+        input (Variable): the detection results, which is a LoDTensor with shape
```

Comment:
the --> The ? same as follows.

Reply:
Done. From http://www.paddlepaddle.org/docs/develop/api/en/fluid/layers.html , upper case is betther.